packages/cli/test/unit/util/build/import-builders.test.ts

```diff
@@ -146,30 +146,6 @@ describe('importBuilders()', () => {
     }
   );
 
-  it.skipIf(isWindows)(
```

Comment:
Removed this old test because it seems that newer versions of npm do not print deprecation warnings.